lib/commands/local/configure.ts

```diff
@@ -163,6 +169,13 @@ export default class LocalConfigureCmd extends Command {
 			name: 'networkKey',
 			default: data.networkKey,
 		},
+		{
+			message:
+				'Enable development mode? (Open ports and root access - Not for production!)',
+			type: 'confirm',
+			name: 'developmentMode',
+			default: false,
```

Comment:
Bot `local configure` and `os configure` update the `config.json` file in a given `.img` file. However, I gather that `local configure` does NOT use the `sdk.models.os.getConfig` method, instead operating locally without depending on balenaCloud. To that end, it uses these templates updated by this PR. I have tested it and it actually works, how cool! :-)